packages/core-web/src/styles/markbind.css

```diff
@@ -463,3 +463,8 @@ button.copy-btn {
 .navbar.navbar-dark.bg-primary .navbar-nav .current {
     background: rgba(255, 255, 255, 0.05);
 }
+
+.dropdown-current {
```

Comment:
you could use vue loader's deep selectors here to encapsulate it in navbar.vue

https://vue-loader.vuejs.org/guide/scoped-css.html

Reply:
I have moved this block into navbar,vue. Thanks.